app/views/posts/show.html.erb

```diff
@@ -1 +1,36 @@
-<h1>This is an individual post</h1>
+<section class="container">
+  <article class="user-card">
+    <img src="<%= @user.photo %>" class="user-photo" alt="user photo">
+    <div class="user-card-info">
+      <h2><%= @user.name %></h2>
+      <p>Number of posts: <%= @user.posts_count %></p>
+    </div>
+  </article>
+
+  <% if @user.recent_posts.any? %>
+      <% @user.recent_posts.each do |post| %>
+        <div class="post-details">
+          <h2>Post #<%= post.id %></h2>
+          <h3><%= post.title %></h3>
+          <h3><%= post.content %></h3>
+          <p>Comments: <%= post.commentsCounter || 0 %>, Likes: <%= post.likes_count || 0 %></p>
```

Comment:
- [ OPTIONAL ] Great job making the required changes but here I noticed that I can see this page from the given wireframe since in your route there is a little miss so I highly recommend fixing it 💯🔥 I assign it as optional because the previous reviewer didn't mention it 🔥💯🤩

- Here the route http://[::1]:3000/users/1/posts/2
- and the wireframe 
![image](https://github.com/mariegrace31/blogApp/assets/58553711/91a7816b-65e0-4aaa-89ee-f9e594b1d496)



but it should look like that 💯
![image](https://github.com/mariegrace31/blogApp/assets/58553711/d2c450e0-ab3c-4355-bb9c-575475742c55)


- And this route http://[::1]:3000/users/1/posts
![image](https://github.com/mariegrace31/blogApp/assets/58553711/d41d1885-720a-4826-9314-86616dfbdefa)

- but it should be like this
![image](https://github.com/mariegrace31/blogApp/assets/58553711/96d83baa-e1e9-45b4-8541-3b9ff752d75e)